changelogs/fragments/2125-git-config-scope-file.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - git_config - fixed scope ``file`` behaviour, added integraton test for it, marked integration tests as destructive (https://github.com/ansible-collections/community.general/issues/2117).
```

```suggestion
  - git_config - fixed scope ``file`` behaviour and added integraton test for it (https://github.com/ansible-collections/community.general/issues/2117).
```
Adding tests is also not really user-facing, but nice to know.